content/tutorials/manage-the-rippled-server/troubleshooting/diagnosing-problems.md

```diff
@@ -0,0 +1,74 @@
+# Diagnosing Problems with rippled
+
+If you have having problems with `rippled`, the first step is to collect more information to accurately characterize the problem. From there, it can be easier to figure out a root cause and a fix.
+
+If your server does not start, see [rippled Server Won't Start](server-wont-start.html) for a list of some possible causes and fixes.
+
+The remainder of this document suggests steps for diagnosing the problem if your server starts successfully.
+
+## Get the server_info
+
+You can use the commandline to get server status information from the local `rippled` instance. For example:
+
+```
+rippled server_info
+```
+
+The response to this command has a lot of information, which is documented along with the [server_info method][].
```

Comment:
Yes. The `[x method][]` links are supplied by the `rippled-api-links.md` snippet, which is only included if you use Dactyl to build the docs instead of viewing the Markdown files in, say, GitHub.